app/src/main/java/org/ieeeguc/ieeeguc/models/User.java

```diff
@@ -1,5 +1,100 @@
 package org.ieeeguc.ieeeguc.models;
 
+import java.util.Date;
 public class User {
+    private Enum type;
+private  String firstName;
+    private  String LastName;
+    private String email;
+    private Enum gender;
+    private Date birthdate;
+    private String ieeeMembershipID;
+    private int committeeID;
+    private String committeeName;
+
+    public User(Enum type, String firstName, String lastName, String email, Enum gender, Date birthdate, String ieeeMembershipID,
+                int committeeID, String committeeName){
+        this.type=type;
+        this.firstName=firstName;
+        this.LastName=lastName;
+        this.gender=gender;
+        this.birthdate=birthdate;
+        this.email=email;
+        this.ieeeMembershipID=ieeeMembershipID;
+        this.committeeID = committeeID;
+        this.committeeName=committeeName;
+
+    }
+    public Enum getType() {
+        return type;
+    }
+
+    public void setType(Enum type) {
+        this.type = type;
+    }
+
+    public String getFirstName() {

```

Comment:
remove all the setters as the attributes will be changed within this class only.
add the getters for the newly added attributes